src/content/docs/en/reference/configuration-reference.mdx

```diff
@@ -512,6 +516,28 @@ To rename the `_astro` path, specify a new directory in `build.assets`.
 }
 ```
 
+<p>
+**If use `Record<string, string>`**
+</p>
+
+Applicable to multiple CDN scenarios. If you static file use different CDN, you can configure it like the example below.
+
+For example, if static file is js file, assets prefix use `https://js.cdn.example.com`. If static file is css file, assets prefix use `https://css.cdn.example.com`. Other assets prefix use `https://cdn.example.com`.
```

Comment:
```suggestion
For example, if the static file is a `.js` file, it will be served from `https://js.cdn.example.com`. If it is a `.css` file, it will be served from `https://css.cdn.example.com`. All other assets will be served from `https://cdn.example.com`.
```
Some refactoring